### DISCUSSION TITLE
DAO Coins: The Future of Fundraising on the Internet

### DISCUSSION THREAD

**Discussion by diamondhands0** [deso-protocol/dips#188] [Large CIPs]
Creating this thread so that we can discuss a new DIP I'm going to be submitting today :)

**Comment by tijno** [comment]
Looking forward to this and in particular get confirmation that deso will "Eat its own dog-food" i hope ;)

**Reply by brootle** [reply]
@tijno can you clarify? I didn't understand about dog food, English is not my native languages, so maybe I missed something? thanks!

**Reply by dgsus** [reply]
> @tijno can you clarify? I didn't understand about dog food, English is not my native languages, so maybe I missed something? thanks!

think he means if the foundation will use the dao

**Comment by diamondhands0** [comment]
Hey all, I'm actually going to close out this discussion and recommend that we discuss in the PR for the DIP here:
* https://github.com/deso-protocol/dips/pull/189
